odl/tomo/util/testutils.py

```diff
@@ -21,8 +21,9 @@
     import pytest
 
 except ImportError:
-    # Use the identity decorator (default of OptionalArgDecorator)
-    from odl.util import OptionalArgDecorator as identity
+
+    def identity(f, **kwargs):
```

Comment:
And there is no other way to do this properly? This feels a bit like a hack.

Reply:
I'm not sure what you mean by "properly". We should have a function that can be used as a decorator and that does nothing. Actually, the `**kwargs` is wrong, there should be a little indirection.

Reply:
I changed the code to accommodate for optional args. Both of these will work and keep `f` untouched:
```python
@identity
def f(x):
    return x

@identity(a=42)
def f(x):
    return x
```

Reply:
What I mean by properly is fixing the import :)

Reply:
There is no way to do that? E.g. just reorder and put testutils last?

Reply:
Maybe, but that import was overkill anyway. I'll leave it as it is currently.